Source/WebCore/html/HTMLFormControlElement.cpp

```diff
@@ -423,69 +423,69 @@ RefPtr<Element> HTMLFormControlElement::invokeTargetElement() const
     if (!canInvoke(*this))
         return nullptr;
 
-    return getElementAttribute(invoketargetAttr);
+    return getElementAttribute(commandforAttr);
 }
 
 constexpr ASCIILiteral togglePopoverLiteral = "togglepopover"_s;
 constexpr ASCIILiteral showPopoverLiteral = "showpopover"_s;
 constexpr ASCIILiteral hidePopoverLiteral = "hidepopover"_s;
 constexpr ASCIILiteral showModalLiteral = "showmodal"_s;
 constexpr ASCIILiteral closeLiteral = "close"_s;
-InvokeAction HTMLFormControlElement::invokeAction() const
+CommandType HTMLFormControlElement::commandType() const
 {
-    auto action = attributeWithoutSynchronization(HTMLNames::invokeactionAttr);
+    auto action = attributeWithoutSynchronization(HTMLNames::commandAttr);
     if (action.isNull() || action.isEmpty())
-        return InvokeAction::Auto;
+        return CommandType::Invalid;
 
     if (equalLettersIgnoringASCIICase(action, togglePopoverLiteral))
-        return InvokeAction::TogglePopover;
+        return CommandType::TogglePopover;
 
     if (equalLettersIgnoringASCIICase(action, showPopoverLiteral))
-        return InvokeAction::ShowPopover;
+        return CommandType::ShowPopover;
 
     if (equalLettersIgnoringASCIICase(action, hidePopoverLiteral))
-        return InvokeAction::HidePopover;
+        return CommandType::HidePopover;
 
     if (equalLettersIgnoringASCIICase(action, showModalLiteral))
-        return InvokeAction::ShowModal;
+        return CommandType::ShowModal;
 
     if (equalLettersIgnoringASCIICase(action, closeLiteral))
-        return InvokeAction::Close;
+        return CommandType::Close;
 
     if (action.contains('-'))
-        return InvokeAction::Custom;
+        return CommandType::Custom;
 
-    return InvokeAction::Invalid;
+    return CommandType::Invalid;
 }
 
-void HTMLFormControlElement::handleInvokeAction()
+void HTMLFormControlElement::handleCommand()
 {
-    RefPtr invokee = invokeTargetElement();
+    RefPtr invokee = commandForElement();
```

Comment:
`target` would be a clearer variable name here.

Reply:
I'll be sure to include this in the follow up PR.